.github/workflows/python-app.yml

```diff
@@ -35,7 +35,7 @@ jobs:
         flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
     - name: Test with pytest
       env:
-        DIMAGIQA_URL: ${{ secrets.DIMAGIQA_URL }}
+        DIMAGIQA_URL: ${{ github.event.client_payload.environment == 'production' && 'https://www.commcarehq.org/accounts/login/' || 'https://staging.commcarehq.org/' }}
```

Comment:
@millerdev Does this look fine?

Reply:
I merged your suggestion, but its [failing](https://github.com/dimagi/dimagi-qa/pull/40/checks?check_run_id=3951348647) 
Is it because it cannot find the URL on push/pull activities?